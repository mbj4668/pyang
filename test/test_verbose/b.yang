module b {
  namespace "urn:b";
  prefix b;
}
