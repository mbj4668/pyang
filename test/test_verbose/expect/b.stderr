# module search path: .:_ROOT_/modules_SYSPATH_
# read b.yang (CL)
